### ISSUE TITLE
after the last update

### ISSUE THREAD

**Issue by itulo** [atomicals/atomicals-js#15]
Today I pulled master and this started happening. Last time I pulled was about 24 hours before.

MacBook Pro Sonoma 14.0
```
~/atomicals-js (master) $ npm run cli get-realm phrase

> atomicals-js@0.1.11 cli
> node dist/cli.js get-realm phrase

{
  "success": false,
  "message": "TypeError: Cannot read properties of undefined (reading 'on')",
  "error": {}
}
/Users/****/atomicals-js/node_modules/rpc-websockets/node_modules/ws/lib/websocket.js:675
      throw new SyntaxError(`Invalid URL: ${address}`);
            ^

SyntaxError: Invalid URL:
    at initAsClient (/Users/***/atomicals-js/node_modules/rpc-websockets/node_modules/ws/lib/websocket.js:675:13)
    at new WebSocket (/Users/***/atomicals-js/node_modules/rpc-websockets/node_modules/ws/lib/websocket.js:85:7)
    at Client._default [as webSocketFactory] (/Users/***/atomicals-js/node_modules/rpc-websockets/dist/lib/client/websocket.js:21:10)
    at Client._connect (/Users/***/atomicals-js/node_modules/rpc-websockets/dist/lib/client.js:406:26)
    at new CommonClient (/Users/***/atomicals-js/node_modules/rpc-websockets/dist/lib/client.js:102:34)
    at Client._createSuperInternal (/Users/***/atomicals-js/node_modules/rpc-websockets/dist/index.js:32:317)
    at new Client (/Users/***/atomicals-js/node_modules/rpc-websockets/dist/index.js:72:19)
    at ElectrumApi.<anonymous> (/Users/***/atomicals-js/dist/api/electrum-api.js:26:23)
    at Generator.next (<anonymous>)
    at /Users/***/atomicals-js/dist/api/electrum-api.js:8:71

Node.js v20.8.0
```

Same happens with `yarn`

**Comment by atomicals** [comment]
Fixed in version 0.1.13

**Comment by itulo** [comment]
Still happening.
